Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/BacklinkObj.diagnostics.cs

```diff
@@ -0,0 +1,14 @@
+﻿[
```

Comment:
Why are these `.cs` files? They look like json

Reply:
This is a limitation of source generators (there is a comment about it in `DiagnosticsEmitter`), because at the moment they can only emit code files, so .cs is added on all generated files. There's a discussion about supporting this https://github.com/dotnet/roslyn/issues/57608. 
Given that this is just for internal use I didn't worried about it too much.